### DISCUSSION TITLE
custom orders with custom resources

### DISCUSSION THREAD

**Discussion by 1gzify** [Paymenter/Paymenter#221] [Ideas]
make it so people can order server with custom resources what they want. They will have to submit it and the admin will have to set a price for what the customer is asking for. and he clicks Submit and it will send an email to the customer who wanted custom resources and it will say the price and he can buy it.

**Comment by 09han09** [comment]
This already exists and is fully automated, it is called configuratie options

**Reply by 1gzify** [reply]
thanks.
